miniapps/shifted/extrapolate.cpp

```diff
@@ -0,0 +1,219 @@
+// Copyright (c) 2010-2021, Lawrence Livermore National Security, LLC. Produced
+// at the Lawrence Livermore National Laboratory. All Rights reserved. See files
+// LICENSE and NOTICE for details. LLNL-CODE-806117.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability visit https://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the BSD-3 license. We welcome feedback and contributions, see file
+// CONTRIBUTING.md for details.
+//
+//            ------------------------------------------------
+//            Extrapolation Miniapp: PDE-based extrapolation
+//            ------------------------------------------------
+//
+// This miniapp extrapolates a finite element function from a set of elements
+// (known values) to the rest of the domain. The set of elements that contains
+// the known values is specified by the positive values of a level set
+// Coefficient. The known values are not modified. The miniapps supports two
+// PDE-based approaches [1, 2], both of which rely on solving a sequence of
+// advection problems in the direction of the unknown parts of the domain.
+// The extrapolation can be constant (1st order), linear (2nd order), or
```

Comment:
Should this comment be in agreement with the `ex_order` input? i.e., 0th/1st/2nd order?

Reply:
Changed the variable names to `*degree` instead of `*order`. Fixed the description text too.